src/TodoListItem.js

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 
-function TodoListItem(props){
-    return(<li >{props.todo.title}</li>);
+function TodoListItem({ todo, onRemoveTodo }) {
+    function removeFunc() {
```

Comment:
Let's reconsider our naming of this function. It's not necessary to label it 'Func' since we know that it's a function because of how were are using it. `Remove` is a good name but we can also add some more meaning by specifying what we are removing. `handleRemove` is also a good option that's widely used.